src/commands.rs

```diff
@@ -1,6 +1,10 @@
 use std::{io::Write, path::Path};
 
-use crate::{file::file_exists, utils::DisplayEnv};
+use crate::{
+    file::{self, file_exists},
+    utils::DisplayEnv,
+};
+use bunt::println as print;
```

Comment:
Should have done this earlier 😅

Reply:
I have just done this in one file just to showcase to you , later you can do same in other files... its long words to type on each print with bunt